src/lib/viewers/controls/media/MediaSettingsMenuSubtitles.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import Settings, { Menu } from '../settings';
+
+export const SUBTITLES_OFF = -1;
```

Comment:
Since this is used by more than just the Menu components, should it live in a more general location, like a constants file?